_sources/Iteration/Listsandforloops.rst

```diff
@@ -60,7 +84,7 @@ The values [0,1,2,3] were provided to make the loop body execute 4 times.
 We could have used any four values.  For example, consider the following program.
 
 
-.. activecode:: ac6_5_3
+.. activecode:: ac6_5_4
```

Comment:
We can't just move ids around like this.  There are people who have outstanding assignments and the autograder relies on these identifiers.

Reply:
Fair enough; will get rid of the changes to unique ids.